pkg/registry/minion/rest.go

```diff
@@ -143,15 +143,15 @@ func ResourceLocation(getter ResourceGetter, connection client.ConnectionInfoGet
 	node := nodeObj.(*api.Node)
 	host := node.Name // TODO: use node's IP, don't expect the name to resolve.
 
-	if portReq != "" {
```

Comment:
Couldn't make out if this was just a mistake or we actually want a whitelist of ports for the transport?